tests/php/UtilsTest.php

```diff
@@ -6,6 +6,23 @@
 
 class UtilsTest extends TestCase {
 
+	/**
+	 * Set up with WP_Mock
+	 *
+	 * Set up common mocks required for multiple tests.
+	 *
+	 * @since x.x.x
+	 */
+	public function setUp(): void {
+		parent::setUp();
+
+
+		// Return voids.
+		\WP_Mock::userFunction( '_prime_post_caches' );
+		\WP_Mock::userFunction( 'update_object_term_cache' );
+		\WP_Mock::userFunction( 'update_postmeta_cache' );
+	}
+
```

Comment:
I tried to include these in `tests/php/includes/TestCase.php` but it wasn't been registered.  I renamed the class in that file to `IsThisEvenUsed` and phpunit didn't throw any fatal errors, so I suspect it isn't used.